api/src/main/java/com/github/eyefloaters/console/api/model/MemberDescription.java

```diff
@@ -1,62 +1,68 @@
 package com.github.eyefloaters.console.api.model;
 
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@JsonInclude(value = Include.NON_NULL)
 public class MemberDescription {
```

Comment:
I'll add a doc comment regarding the consumer group. I hesitate to change more than that until we see what queues look like. For example, it may be that we could simply make this a parameterized type `MemberDescription<T>` where `T` is the type of `assignments`, which may differ for queues. Obviously that's all just guessing at this point, but we can make a more informed decision in the future I think.